frameworks/Java/armeria/src/main/java/hello/services/HelloService.java

```diff
@@ -0,0 +1,57 @@
+package hello.services;
+
+import hello.models.Message;
+
+import java.nio.charset.StandardCharsets;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+import com.linecorp.armeria.common.HttpData;
+import com.linecorp.armeria.common.HttpHeaderNames;
+import com.linecorp.armeria.common.HttpHeaders;
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.HttpStatus;
+import com.linecorp.armeria.common.MediaType;
+import com.linecorp.armeria.server.annotation.Get;
+import com.linecorp.armeria.server.annotation.ProducesJson;
+
+public class HelloService
+{
+  private static final byte[]       PLAINTEXT      = "Hello, World!".getBytes(
+      StandardCharsets.UTF_8);
+  private static final ObjectMapper MAPPER         = new ObjectMapper();
+  private static final DateFormat   DATE_FORMATTER = new SimpleDateFormat(
+      "E, dd MMM yyyy HH:mm:ss z");
+
+  @Get("/plaintext") public HttpResponse plaintext()
+  {
+    HttpHeaders headers = HttpHeaders.of(HttpStatus.OK).add(
```

Comment:
See my previous comment about line breaks near the `annotatedService(...)` method calls.  The same advice applies here.